### PR TITLE
ADEN-3723 Liftium cleanup

### PR DESCRIPTION
**Pull request by nandy-andy**
We're in the process of removing Liftium. We don't need it in tests as well ;)

Connected PRs:
* https://github.com/Wikia/app/pull/11010,
* https://github.com/Wikia/jenkins-jobs/pull/75.

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'java.util.regex.Matcher'.](https://github.com/Wikia/selenium-tests/blob/59f2e014fde899382c867816ef597dbdaed16cc4/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java#L31) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'java.util.regex.Pattern'.](https://github.com/Wikia/selenium-tests/blob/59f2e014fde899382c867816ef597dbdaed16cc4/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java#L32) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAdDriverForcedStatusOasis.java [hunk 1]
```diff
@@ -8,19 +8,21 @@
...
-public class TestAdDriverForcedStatus extends TemplateNoFirstLoad {
+public class TestAdsAdDriverForcedStatusOasis extends TemplateNoFirstLoad {
 
   @Test(
       dataProviderClass = AdsDataProvider.class,
       dataProvider = "adDriverForcedStatusSuccess",
-      groups = {"TestAdDriverForcedStatusSuccess_GeoEdgeFree", "Ads"}
+      groups = {
+          "AdsAdDriverForcedStatusOasis",
+          "Ads"
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAdDriverForcedStatusOasis.java]: Consider removing `Ads` group and then you can write `groups = "AdsAdDriverForcedStatusOasis"`

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAdDriverForcedStatusOasis.java [hunk 2]
```diff
@@ -8,19 +8,21 @@
...
+      }
   )
-  public void TestAdDriverForcedStatusSuccess_GeoEdgeFree(String wikiName,
+  public void adsAdDriverForcedStatusSuccessOasis(String wikiName,
                                                           String article,
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAdDriverForcedStatusOasis.java]: Fix indentation

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsRoadblocksAfterMultiplePageViewsOasis.java [hunk 3]
```diff
@@ -7,16 +7,16 @@
...
 
   @Test(
       dataProviderClass = AdsDataProvider.class,
       dataProvider = "skinLimited",
-      groups = "TestRoadblock_GeoEdgeFree"
+      groups = "AdsTestRoadblocksOasis"
   )
-  public void TestRoadblock_GeoEdgeFree(String wikiName, String article, Dimension windowResolution,
+  public void adsTestRoadblockOasis(String wikiName, String article, Dimension windowResolution,
                                         String expectedAdSkinLeftPartPath,
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsRoadblocksAfterMultiplePageViewsOasis.java]: Fix indentation

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAdDriverForcedStatusOasis.java (hunk 1 above)
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAdDriverForcedStatusOasis.java]: Sure, it seems there is no usage of `Ads` group in jenkins jobs repo.
```
$ git grep -in "group: Ads"
```

**Comment by drets** [comment]
LGTM :+1:

**Pull request merged**